tests/run-make/split-debuginfo/Makefile

```diff
@@ -237,7 +237,7 @@ else # Some non-Windows, non-Darwin platforms are not stable, and some are.
 	unpacked-lto: unpacked-lto-split unpacked-lto-single
 
 	# - rmeta file added to rlib, no object files are generated and thus no debuginfo is generated
-	# - `.o` present (bitcode)
+	# - `.o` not present
```

Comment:
Local testing reveals this comment is wrong, which is corroborated by the sentence immediately above it.